Docs 1935 (#2828)
viamrobotics/docs: docs/tutorials/services/data-mlmodel-tutorial.md

To deploy a model to your machine:

1. Navigate to the **CONFIGURE** tab of your machine's page in [the Viam app](https://app.viam.com).
1. Click the **+** icon next to your machine part in the left-hand menu and select **Service**.
1. Select the `ML model` type, then select the `TFLite CPU` model.
1. Enter a name or use the suggested name, like `my-mlmodel-service`, for your service and click **Create**.

> 1. In the resulting ML Model service configuration pane, select **Deploy model on machine**, then select the model you just trained from the **Models** dropdown menu.

Reviewer: ```suggestion
1. In the resulting ML Model service configuration pane, select **Deploy model on machine**, then select the model you just trained from the **Select model** dropdown menu.
```

   {{< imgproc src="/tutorials/data-management/mlmodel-service-conf.png" alt="The ML model service configuration pane showing the required settings to deploy the my-classifier-model." resize="600x" >}}

> 1. Click **Save Config** at the bottom of the window to save your changes.

Reviewer: ```suggestion
1. Click **Save** at the top right of the window to save your changes.
```

### Add the vision service

1. Click the **+** icon next to your machine part in the left-hand menu and select **Service**.
1. Select the `vision` type, then select the `ML model` model.
1. Enter a name, like `my-vision-service`, or use the suggested name for your service and click **Create**.

> 1. Select the ML model service your model is deployed on from the **ML Model** dropdown.

Reviewer: Should delete either 247 or 248-- I have no preference

> 1. Select the ML model service your model is deployed on from the **ML Model** dropdown.

Author: ```suggestion
```
